internal/vds/vds.cpp

```diff
@@ -419,7 +433,8 @@ struct vdsbuffer fetch_fence(
     const std::string& credentials,
     const std::string& coordinate_system,
     const float* coordinates,
-    size_t npoints
+    size_t npoints,
+    enum interpolationMethod interpolationMethod
```

Comment:
This is already a cpp file and chosen convention is a default snake case (coordinate_system). So interpolation_method is better.